beacon_node/network/src/sync/block_lookups/single_block_lookup.rs

```diff
@@ -307,19 +328,19 @@ impl<E: EthSpec> BlobRequestState<E> {
     }
 }
 
-/// The state of the block request component of a `SingleBlockLookup`.
+/// The state of the blob request component of a `SingleBlockLookup`.
```

Comment:
```suggestion
/// The state of the custody request component of a `SingleBlockLookup`.
```

Reply:
Fixed in 96e2363d721f55d79f66f98f15fd61a854c7dc62